docs/02-app/01-building-your-application/01-routing/04-linking-and-navigating.mdx

```diff
@@ -263,7 +263,7 @@ Learn more about how the [Router Cache](/docs/app/building-your-application/cach
 
 Partial rendering means only the route segments that change on navigation re-render on the client, and any shared segments are preserved.
 
-For example, when navigating between two sibling routes, `/dashboard/settings` and `/dashboard/analytics`, the `settings` and `analytics` pages will be rendered, and the shared `dashboard` layout will be preserved.
+For example, when navigating between two sibling routes, `/dashboard/settings` and `/dashboard/analytics`, the `settings` and `analytics` pages will be fully rendered (pages will unmount and state will be lost), and the shared `dashboard` layout will be preserved.
```

Comment:
```suggestion
For example, when navigating between two sibling routes, `/dashboard/settings` and `/dashboard/analytics`, the `settings` page will be unmounted, the `analytics` page will be mounted with fresh state, and the shared `dashboard` layout will be preserved.
```

Reply:
@ijjk Thank you for the suggestion I've applied it. But now I think that it only partially says what I wanted to share here - that this will also be true for Dynamic Segments. Maybe at the end add something like: `The same will happen when navigating between Dynamic Segments. `

Reply:
That should not be the case with a dynamic segment as long as it's the same, unless you're trying to stress different dynamic segments?

Reply:
It is the case, you can read more about that in discussion I've linked here - https://github.com/vercel/next.js/discussions/50711


> Currently, when navigating client-side from one path to another within the same dynamic route segment, e.g. from /en to /de, the layout router will cause all components on the page to get unmounted and remounted.
> ...
> With the above example, navigating client-side from /en to /de will cause all components on the page to get unmounted and remounted, even though we remain in the same dynamic segment ([lang]) and on the same page. This is due to the cache key including the dynamic value, i.e. changing from lang|en|d to lang|de|d.
>

Reply:
@ijjk

Reply:
Thank you, added changes according to your suggestion, and I believe this PR can be checked and merged now.